webrtc.html

```diff
@@ -6112,6 +6406,75 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
         </tr>
       </tbody>
     </table>
+
+    <p>The following events fire on <code><a>RTCIceTransport</a></code>
+    objects:</p>
+
+    <table>
+      <tr>
+        <th>Event name</th>
+
+        <th>Interface</th>
+
+        <th>Fired when...</th>
+      </tr>
+
+      <tbody>
+        <tr>
+          <td><dfn id=
+          "event-icetransport-statechange"><code>statechange</code></dfn></td>
```

Comment:
you need to add an attribute `data-lt='icetransport-statechange'` otherwise respec will complains about duplicate definitions for `statechange`